web-ui/src/context/util.js

```diff
@@ -18,20 +19,19 @@ export const sortAnniversaries = anniversaryData => {
   return anniversaryData.sort((a, b) => b.yearsOfService - a.yearsOfService);
 };
 
+function monthDayToDayOfYear(monthDayString) {
+  const [month, day] = monthDayString.split('/');
+  const now = new Date();
+  const date = new Date(now.getFullYear(), Number(month) - 1, Number(day));
+  return getDayOfYear(date);
+}
+
 export const sortBirthdays = birthdayData => {
-  return birthdayData.sort(
```

Comment:
I changed the logic of this to sort on "day of year" values and that is producing the correct sort order.